ui/lib/replication/addon/controllers/application.js

```diff
@@ -2,22 +2,23 @@ import { isPresent } from '@ember/utils';
 import { alias } from '@ember/object/computed';
 import { inject as service } from '@ember/service';
 import Controller from '@ember/controller';
+import { copy } from 'ember-copy';
+import { resolve } from 'rsvp';
 
 const DEFAULTS = {
   token: null,
   id: null,
   loading: false,
   errors: [],
-  showFilterConfig: false,
   primary_api_addr: null,
   primary_cluster_addr: null,
   filterConfig: {
-    mode: 'whitelist',
+    mode: null,
     paths: [],
   },
 };
 
-export default Controller.extend(DEFAULTS, {
+export default Controller.extend(copy(DEFAULTS, true), {
```

Comment:
why do we need a deep copy of DEFAULTS here?

Reply:
I think this was probably a bug before, but wasn't as obvious because of the way the paths were rendered (they were always listed, but checked or unchecked). With the new component, they're only rendered if they're in the list and we're mutating the defaults when we push on to the `paths` array. The deep copy makes sure we get a new array every time the controller is reset.